src/nft/components/explore/Banner.tsx

```diff
@@ -117,7 +117,6 @@ const TRENDING_COLLECTION_SIZE = 5
 
 const Banner = () => {
   const navigate = useNavigate()
-  const isMobile = useIsMobile()
```

Comment:
need to remove the import as well

Reply:
rule number one: never do inline Github tweaks lol